scripts/data_files/query_config.fmt

```diff
@@ -102,6 +101,7 @@
 #include "mbedtls/x509_crt.h"
 #include "mbedtls/x509_csr.h"
 #include "mbedtls/xtea.h"
+#include "test/certs.h"
```

Comment:
Note: reminder to remove this from the `query_config.fmt` as well as the previous, `query_config.c`.